layouts/partials/comments/provider/waline.html

```diff
@@ -19,7 +19,7 @@
 {{- $replaceKeys := dict "serverurl" "serverURL" "requiredmeta" "requiredMeta" "wordlimit" "wordLimit" "pagesize" "pageSize" "imageuploader" "imageUploader" "texrenderer" "texRenderer" -}}
 
 {{- range $key, $val := . -}}
-    {{- if $val -}}  
+    {{- if or $val (eq $val false) -}}  

```

Comment:
Thanks!

I think it might be better to use `isset` to check. (https://gohugo.io/functions/isset/)

Reply:
Never mind, I'm trying to replace it with `isset` and it does not work as I expected. I'll try it again later.

Reply:
My proposal is:

```diff
- {{- if or $val (eq $val false) -}}
+ {{- if ne $val nil -}}
```

The reason for this is that Hugo treats the following values as `false`:

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/5889006/999f6d98-5931-4456-b5db-ead80d0a3a79)

Adding `eq $val false` does not cover all these cases. For example, setting `wordLimit: 0` would cause the same bug because `0` is falsy.

The check we need is whether _the field is `null` or not_. What do you think?

Reply:
Nice comments. Agree with it.